src/Maestro/Maestro.Data/Migrations/BuildAssetRegistryContextModelSnapshot.cs

```diff
@@ -307,6 +307,21 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.ToTable("DependencyFlowEvents");
                 });
 
+            modelBuilder.Entity("Maestro.Data.Models.GoalTime", b =>
+                {
+                    b.Property<int>("DefinitionId");
+
+                    b.Property<int>("ChannelId");
+
+                    b.Property<int>("Minutes");
+
+                    b.HasKey("DefinitionId", "ChannelId");
+
+                    b.HasIndex("ChannelId");
```

Comment:
@alexperovich  hasIndex was part of the migration I removed. Looks like this is still here. Snapshot is not getting refreshed. Can I delete this one line and try again? Or is this a different process?

Reply:
If you `git checkout master -- src/Maestro/Maestro.Data/Migrations/BuildAssetRegistryContextModelSnapshot.cs`

Reply:
Wont that take out my current change for foreign key? So I add the migration again ?

Reply:
Yea, just reset everything in the Migrations folder and re-make the migration.